_query-dsl/query-dsl/minimum-should-match.md

```diff
@@ -0,0 +1,450 @@
+        {
+          "match": {
+            "text": "two"
+          }
+        },
+        {
+          "match": {
+            "text": "three"
+          }
+        },
+        {
+          "match": {
+            "text": "four"
+          }
+        }
+      ],
+      "minimum_should_match": "80%"
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Because `minimum_should_match` is `80%`, the number of optional clauses to match is calculated as 4 &middot; 0.8 = 3.2 and then rounded down to 3. Therefore, the results contain documents that match at least 3 clauses:
+
+```json
+{
+  "took": 40,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 2,
+      "relation": "eq"
+    },
+    "max_score": 2.494999,
+    "hits": [
+      {
+        "_index": "testindex",
+        "_id": "4",
+        "_score": 2.494999,
+        "_source": {
+          "text": "one two three four OpenSearch"
+        }
+      },
+      {
+        "_index": "testindex",
+        "_id": "3",
+        "_score": 1.5744598,
+        "_source": {
+          "text": "one two three OpenSearch"
+        }
+      }
+    ]
+  }
+}
+```
+
+Now let's specify `minimum_should_match` as `-20%`:
+
+```json
+GET testindex/_search
+{
+  "query": {
+    "bool": {
+      "must": [
+        {
+          "match": {
+            "text": "OpenSearch"
+          }
+        }
+      ], 
+      "should": [
+        {
+          "match": {
+            "text": "one"
+          }
+        },
+        {
+          "match": {
+            "text": "two"
+          }
+        },
+        {
+          "match": {
+            "text": "three"
+          }
+        },
+        {
+          "match": {
+            "text": "four"
+          }
+        }
+      ],
+      "minimum_should_match": "-20%"
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The number of non-matching optional clauses that a document can have is calculated as 4 &middot; 0.2 = 0.8 and rounded down to 0. Thus, the results contain only one document that matches all optional clauses:
+
+```json
+{
+  "took": 41,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 1,
+      "relation": "eq"
+    },
+    "max_score": 2.494999,
+    "hits": [
+      {
+        "_index": "testindex",
+        "_id": "4",
+        "_score": 2.494999,
+        "_source": {
+          "text": "one two three four OpenSearch"
+        }
+      }
+    ]
+  }
+}
+```
+
+Note that specifying a positive percentage (`80%`) and negative percentage (`-20%`) did not result in the same number of optional clauses a document must match because in both cases the result was rounded down. If the number of optional clauses were, for example, 5, then both `80%` and `-20%` would have produced the same number of optional clauses a document must match (4).
```

Comment:
```suggestion
Note that specifying a positive percentage (`80%`) and negative percentage (`-20%`) did not result in the same number of optional clauses a document must match because, in both cases, the result was rounded down. If the number of optional clauses were, for example, 5, then both `80%` and `-20%` would have produced the same number of optional clauses a document must match (4).
```